include/mbedtls/dhm.h

```diff
@@ -23,7 +23,13 @@
 #ifndef MBEDTLS_DHM_H
 #define MBEDTLS_DHM_H
 
+#if !defined(MBEDTLS_CONFIG_FILE)
+#include "config.h"
+#else
+#include MBEDTLS_CONFIG_FILE
+#endif
 #include "bignum.h"
+#if !defined(MBEDTLS_DHM_ALT)
```

Comment:
This guards the `extern "C" { ...` path [below](https://github.com/ARMmbed/mbedtls/pull/1239/files#diff-12e227586f72d7fc03d416921e47b43aR148) but not its closing `}`. I'd suggest to proceed as in [sha1.c](https://github.com/ARMmbed/mbedtls/blob/development/include/mbedtls/sha1.h).

Reply:
You are right!
Thanks.